### PR TITLE
chore(signup): fix up signup page + weekdayToString bugfix

### PR DESCRIPTION
**Pull request by MatoPlus**
Fix css of signup page + other small things like weekdayToString

### Notion ticket link

[Fix click to outline selected child in sign-up page](https://www.notion.so/uwblueprintexecs/Fix-click-to-outline-selected-child-in-sign-up-page-6b2697e0c821438f897b5538dd8e3a02)
[Fix weekdayToString unreachable code](https://www.notion.so/uwblueprintexecs/Fix-weekdayToString-unreachable-code-207900dbc221474aabdcfe94ae22dcb9)

https://user-images.githubusercontent.com/44826218/135784598-6bc88bc1-7436-4cd3-9b38-084b3e2f8519.mp4

### Implementation description

- Finish touchup on signup page (select highlight)
- Fix switch statements of weekdayToString util

### Steps to test

1. go to `/signup`
2. next button should be disabled
3. selected option should pop with blue text/border
4. next is enabled

### What should reviewers focus on?

- Dev: Code style of signup.tsx
- Design: Workflow + Look & feel of signup page

### Checklist

-   [X] My PR name is descriptive and in imperative tense
-   [X] I have run the linter
-   [X] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by gregmaxin — APPROVED** [review]
LGTM!

File: pages/signup.tsx [hunk 1]
```diff
@@ -27,46 +37,67 @@ export default function Signupform(): JSX.Element {
...
+                                Volunteer
+                            </Text>
                         </Button>
                     </Center>
                     <Center>
                         <Button
-                            backgroundColor="brand.500"
-                            color="brand.100"
+                            _hover={{
+                                backgroundColor: isEmpty()
```
Comment (feng-eric) [inline pages/signup.tsx]: nit: It's a bit hard to tell what `isEmpty` is for by the function name.

**Review by feng-eric — APPROVED** [review]
LGTM other than 1 nitpick comment I had!

**Review by bekahma — APPROVED** [review]
approved from design!

**Pull request merged**